redux/redux.d.ts

```diff
@@ -4,46 +4,77 @@
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 declare module Redux {
+    // FSA-compliant action (from redux-actions.d.ts)
+    // See: https://github.com/acdlite/flux-standard-action
+    type Action = {
+        type: string
+        payload?: any
+        error?: boolean
+        meta?: any
+    };
 
-    interface ActionCreator extends Function {
-        (...args: any[]): any;
+	// Not used...?
+	interface ActionCreator extends Function {
+        (...args: any[]): Action;
     }
 
-    interface Reducer extends Function {
-        (state: any, action: any): any;
+	// Also not used here...
+	interface StoreMethods<T,A extends Action> {
+        dispatch: Dispatch;
+        getState(): T;
     }
 
-    interface Dispatch extends Function {
-        (action: any): any;
+	// Reducer is a function that takes a state of type T and an
+	// action and returns a new state, also of type T
+    interface Reducer<T> extends Function {
+        (state: T, action: any): T;
```

Comment:
you could use the Action type here rather than any, if you make it less restrictive than the FSA